docs/sources/tutorials/get-started.md

```diff
@@ -138,15 +138,16 @@ loki.source.file "log_scrape" {
 
 This configuration creates a [loki.source.file][] component named `log_scrape`, and shows the pipeline concept of {{< param "PRODUCT_NAME" >}} in action. The `log_scrape` component does the following:
 
-1. It connects to the `local_files` component (its "source" or target).
-1. It forwards the logs it scrapes to the "receiver" of another component called `filter_logs` that you will define next.
-1. It provides extra attributes and options, in this case, you will tail log files from the end and not ingest the entire past history.
+1. It connects to the `local_files` components as its "source" or target.
```

Comment:
I think `component` here is correct; there's only one component labelled `local_files`.